### PR TITLE
Let's use const.

### PR DESCRIPTION
**Pull request by rebornix**
var --> const.

❤️ 👍

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
```
Comment (rebornix) [inline app.js]: This is great.

File: app.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
-var express = require('express');
-var path = require('path');
```
Comment (auchenberg) [inline app.js]: not used, remove.

Comment (auchenberg) [reply app.js]: thanks ts.

Comment (auchenberg) [inline app.js]: how about new one.

Comment (auchenberg) [inline app.js]: Unsued variable. Please remove

File: app.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
 
-var hbs = require('hbs');
-var session = require('express-session');
+const hbs = require('hbs');
+const session = require('express-session');
```
Comment (rebornix) [inline app.js]: session is not used.

File: app.js [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
 
-var hbs = require('hbs');
```
Comment (rebornix) [inline app.js]: hbs in original file.

File: app.js [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
```
Comment (rebornix) [inline app.js]: hbs in original content.

Comment (rebornix) [inline app.js]: hbs in original content.

File: app.js (hunk 4 above)
Comment (rebornix) [inline app.js]: hbs in original content.

File: app.js (hunk 3 above)
Comment (kieferrm) [reply app.js]: Will **remove** this

**Comment by RMacfarlane** [comment]
These changes look good to me

**Comment by RMacfarlane** [comment]
You can go ahead and merge this

File: app.js [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
-var hbs = require('hbs');
-var session = require('express-session');
+const hbs = require('hbs');
+const session = require('express-session');
 
-var index = require('./routes/index');
+const index = require('./routes/index');
 
-var app = express();
+const app = express();
```
Comment (RMacfarlane) [inline app.js]: const app

File: app.js (hunk 4 above)
Comment (rebornix) [reply app.js]: cool

Comment (rebornix) [reply app.js]: test

Comment (rebornix) [reply app.js]: test2

**Comment by RMacfarlane** [comment]
👍

**Comment by RMacfarlane** [comment]
🌵

**Comment by RMacfarlane** [comment]
test

**Comment by RMacfarlane** [comment]
test

File: app.js (hunk 6 above)
Comment (RMacfarlane) [reply app.js]: test

File: app.js [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
 
 // view engine setup
 app.set('views', path.join(__dirname, 'views'));
```
Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [inline app.js]: test

File: app.js (hunk 6 above)
Comment (RMacfarlane) [reply app.js]: reply

Comment (RMacfarlane) [reply app.js]: reply 2

Comment (RMacfarlane) [reply app.js]: reply 3

Comment (RMacfarlane) [reply app.js]: reply 4

File: app.js (hunk 7 above)
Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [reply app.js]: test

Comment (RMacfarlane) [reply app.js]: ?

Comment (RMacfarlane) [reply app.js]: ?

Comment (RMacfarlane) [reply app.js]: test

Comment (RMacfarlane) [reply app.js]: asdf

File: app.js [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
 
-var hbs = require('hbs');
-var session = require('express-session');
+const hbs = require('hbs');
+const session = require('express-session');
 
-var index = require('./routes/index');
+const index = require('./routes/index');
```
Comment (RMacfarlane) [inline app.js]: new comment

Comment (RMacfarlane) [reply app.js]: test

File: app.js (hunk 7 above)
Comment (RMacfarlane) [reply app.js]: test

Comment (RMacfarlane) [reply app.js]: asdf

File: app.js (hunk 4 above)
Comment (rebornix) [reply app.js]: test3 with new dom api.

File: app.js [hunk 9]
```diff
@@ -1,16 +1,16 @@
...
 
-var app = express();
+const app = express();
 
 // view engine setup
```
Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [reply app.js]: test reply

File: app.js [hunk 10]
```diff
@@ -1,16 +1,16 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
```
Comment (RMacfarlane) [inline app.js]: A new comment

Comment (RMacfarlane) [reply app.js]: another comment

File: app.js [hunk 11]
```diff
@@ -1,16 +1,16 @@
...
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
```
Comment (RMacfarlane) [inline app.js]: testing

File: app.js [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
 
-var hbs = require('hbs');
-var session = require('express-session');
+const hbs = require('hbs');
+const session = require('express-session');
 
-var index = require('./routes/index');
+const index = require('./routes/index');
```
Comment (RMacfarlane) [inline app.js]: line 11

Comment (RMacfarlane) [reply app.js]: ?

File: app.js (hunk 1 above)
Comment (RMacfarlane) [inline app.js]: test

File: app.js (hunk 7 above)
Comment (RMacfarlane) [inline app.js]: test

File: app.js (hunk 4 above)
Comment (RMacfarlane) [reply app.js]: ZXc

File: app.js (hunk 9 above)
Comment (RMacfarlane) [inline app.js]: test test

Comment (RMacfarlane) [reply app.js]: test reply

Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [reply app.js]: test test

**Comment by rebornix** [comment]
LGTM, time to merge.

**Comment by rebornix** [comment]
Merge

File: app.js (hunk 7 above)
Comment (RMacfarlane) [inline app.js]: new comment

Comment (RMacfarlane) [reply app.js]: new reply

File: app.js (hunk 9 above)
Comment (RMacfarlane) [inline app.js]: test

File: app.js [hunk 13]
```diff
@@ -1,16 +1,16 @@
...
 
-var app = express();
+const app = express();
```
Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [reply app.js]: test reply

Comment (RMacfarlane) [reply app.js]: comment #3

File: app.js [hunk 14]
```diff
@@ -1,16 +1,16 @@
...
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
```
Comment (RMacfarlane) [inline app.js]: whoops

File: app.js (hunk 11 above)
Comment (RMacfarlane) [inline app.js]: OK

File: app.js (hunk 13 above)
Comment (RMacfarlane) [reply app.js]: another comment

Comment (RMacfarlane) [reply app.js]: comment number 4

Comment (RMacfarlane) [reply app.js]: new comment

Comment (RMacfarlane) [reply app.js]: comment

Comment (RMacfarlane) [reply app.js]: test

File: app.js (hunk 7 above)
Comment (RMacfarlane) [inline app.js]: test

Comment (RMacfarlane) [reply app.js]: test reply

File: app.js (hunk 1 above)
Comment (RMacfarlane) [inline app.js]: in file comment

Comment (RMacfarlane) [reply app.js]: great

File: app.js (hunk 4 above)
Comment (chrisdias) [reply app.js]: hello from universal

**Comment by RMacfarlane** [comment]
test

File: app.js [hunk 15]
```diff
@@ -1,16 +1,16 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
```
Comment (RMacfarlane) [inline app.js]: looks like tihs is unused

**Comment by RMacfarlane** [comment]
a

**Comment by RMacfarlane** [comment]
hello

File: app.js [hunk 16]
```diff
@@ -1,16 +1,16 @@
...
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
 
-var hbs = require('hbs');
-var session = require('express-session');
+const hbs = require('hbs');
```
Comment (RMacfarlane) [inline app.js]: cool

File: app.js (hunk 7 above)
Comment (RMacfarlane) [inline app.js]: test

File: app.js (hunk 6 above)
Comment (RMacfarlane) [inline app.js]: test

File: app.js [hunk 17]
```diff
@@ -1,16 +1,16 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
```
Comment (RMacfarlane) [inline app.js]: test

File: app.js (hunk 9 above)
Comment (RMacfarlane) [inline app.js]: ?

File: app.js (hunk 13 above)
Comment (RMacfarlane) [inline app.js]: asdf

File: app.js (hunk 9 above)
Comment (RMacfarlane) [inline app.js]: hmm

**Pull request closed without merging**